src/installation/guides/chroot.md

```diff
@@ -0,0 +1,313 @@
+# Installation via chroot (x86 or x86_64)
+
+This guide details the process of manually installing Void Linux (the chroot
+method) on an x86 or x86_64 PC architecture. It is assumed that you have a
+familiarity with Linux but not necessarily that you have performed a chroot
+installation before. If you are unfamiliar with the process, this guide can be
+followed literally to create a "typical" setup (one partition is used on one
+disk that is connected by SATA, IDE or USB). If you are comfortable with the
+process, this guide should serve as an outline for reference, where each step
+may be modified to create less typical setups (such as [full disk
+encryption](./fde.md)).
```

Comment:
" ... USB disk. Each step may be modified to create less typical setups, such as \[full disk encryption\](./fde.md)."

Reply:
yes